### ISSUE TITLE
az sentinel alert-rule-action list uses unavailable api version

### ISSUE THREAD

**Issue by Kaloszer** [Azure/azure-cli#24661]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Related command**
az sentinel alert-rule action list -g somerg -w someworkspace --rule-name '/subscriptions/somesubscritpion/resourceGroups/somergsentinel/providers/Microsoft.OperationalInsights/workspaces/someworkspace/providers/Microsoft.SecurityInsights/alertRules/ruleidentifier' --debug

**Describe the bug**

Unable to list alert-rule actions using az-cli, when invoking a request it uses api that is not available, in the request when it is being made (--debug) I can see that it ends with:

![image](https://user-images.githubusercontent.com/62133917/201955945-95e325bd-52a8-4ed8-848b-8e63f3534f49.png)

```

/actions?api-version=2022-06-01-preview

```

![image](https://user-images.githubusercontent.com/62133917/201955225-f80ba61f-c433-4f9e-a51a-582f8f91ef6d.png)
Same with current preview for az alert-rule action 2022-07-01-preview
![image](https://user-images.githubusercontent.com/62133917/201955433-0809cebb-71d8-42dc-acf3-a2f69005e286.png)

**To Reproduce**

Simply create alert rule, add an action on trigger and attempt to list it using mentioned command

**Expected behavior**

Command should list alert rule actions

**Environment summary**

azure-cli                         2.42.0
core                              2.42.0
telemetry                          1.0.8

Extensions:
account                            0.2.5
sentinel                           0.2.0

Dependencies:
msal                              1.20.0
azure-mgmt-resource             21.1.0b1

Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
Extensions directory 'C:\Users\xxxx\.azure\cliextensions'

Python (Windows) 3.10.8 (tags/v3.10.8:aaaf517, Oct 11 2022, 16:37:59) [MSC v.1933 32 bit (Intel)]

Legal docs and information: aka.ms/AzureCliLegal

Your CLI is up-to-date.

**Additional context**

**Comment by Kaloszer** [comment]
Issue also appears in Az/AzPreview (Powershell module) (https://github.com/Azure/azure-powershell/issues/20143) - so there's no way to use CLI to access these at the moment - I guess Invoke-AzRestMethod would be the only way?

**Comment by yonzhan** [comment]
route to CXP team

**Comment by SaurabhSharma-MSFT** [comment]
@Kaloszer I am able to get the alert rule action using Azure CLI 
![image](https://user-images.githubusercontent.com/38112130/206123214-eb8aa65c-7d21-4a51-b77b-213268535314.png)
Instead of passing rule name with resource path, can you please check by just passing the name only.

**Comment by Kaloszer** [comment]
Hey @SaurabhSharma-MSFT - yeah it works with just the sole name, question is if it's not supposed to be possible with the whole identifier? If so this is a feature not a bug and should be closed with a 'won't fix'

**Comment by SaurabhSharma-MSFT** [comment]
@Kaloszer I do not think it is possible as [Actions - List By Alert Rule](https://learn.microsoft.com/en-us/rest/api/securityinsights/stable/actions/list-by-alert-rule?tabs=HTTP&source=docs#code-try-0) REST API doesn't work with the whole resource identifier and throwing similar errors.
![image](https://user-images.githubusercontent.com/38112130/206143340-2a39a3da-0714-45b7-ad95-0621b84c4c13.png)

**Comment by Kaloszer** [comment]
Then I guess it works as designed - it would be nice if the documentation specifically said you cannot use the full identifier but have to use the individual name!

Thanks for investigating
